db/migrate/20170509171615_remove_password_digest_from_user.rb

```diff
@@ -0,0 +1,5 @@
+class RemovePasswordDigestFromUser < ActiveRecord::Migration[5.0]
```

Comment:
Clearance calls it `encrypted_password`.

Should this rename the column instead of dropping it?